tormenta-avro/src/main/scala/com/twitter/tormenta/scheme/avro/AvroScheme.scala

```diff
@@ -19,15 +19,20 @@ package com.twitter.tormenta.scheme.avro
 import com.twitter.bijection.Injection
 import scala.util.{ Failure, Success }
 import com.twitter.tormenta.scheme.Scheme
+import java.nio.ByteBuffer
 
 /**
  * @author Mansur Ashraf
  * @since 9/14/13
  */
 trait AvroScheme[T] extends Scheme[T] {
+  def decodeRecord(buffer: ByteBuffer)(implicit inj: Injection[T, Array[Byte]]): TraversableOnce[T] = {
```

Comment:
`Bufferable` looks awesome, updated.
I'm wondering should I move `BufferableScheme` to tormenta-core (I don't like dependency on bijection from core), leave as now in tormenta-avro (I think this one is the best) or create separate tormenta-bijection (which doesn't sound good for one tiny class IMO). What do you think?